cgotorch/cgotorch.cc

```diff
@@ -22,14 +23,83 @@ char *exception_str(const std::exception &e) {
   return r;
 }
 
+std::unordered_map<std::string, torch::nn::init::FanModeType> fan_mode_map = {
+    {"fan_in", torch::kFanIn},
+    {"fan_out", torch::kFanOut},
+};
+
+std::unordered_map<std::string, torch::nn::init::NonlinearityType>
```

Comment:
It seems that this global variable is only used by `KaimingUniform_`. Should we move it into `KaimingUniform_` as a local static variable?

Reply:
There is another parameter initialization named [`kaiming_normal_`](https://pytorch.org/cppdocs/api/function_namespacetorch_1_1nn_1_1init_1ac8a913c051976a3f41f20df7d6126e57.html).  It also uses these two maps. So let's take it as global variables.